extension/defaults/preferences/prefs.js

```diff
@@ -121,3 +121,7 @@ pref("PersonaService.logging.dump", "None");
 
 // Whether or not the toolbar button has been installed once (Firefox 4 support)
 pref("extensions.personas.toolbarButtonInstalled", false);
+
+pref("extensions.personas.tbadded",false);
+pref("extensions.personas.tbinsert",true);
+pref("extensions.personas.tbplaced",false);
```

Comment:
`tbinsert` controls if the toolbar button should be inserted into UI. Say, if user removes it from UI, the button will not be added to UI anymore by the help of `tbinsert`.

`tbplaced` controls the placement of the toolbar button. At first install, button is placed and preference is saved. So add-on will not try to place it again and again by the help of `tbplaced` preference value.